libs/dh/shared/util-application-insights/src/lib/dh-application-insights.service.ts

```diff
@@ -0,0 +1,67 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Inject, Injectable } from '@angular/core';
+import { Router } from '@angular/router';
+import { AngularPlugin } from '@microsoft/applicationinsights-angularplugin-js';
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+
+import {
+  DhAppEnvironmentConfig,
+  dhAppEnvironmentToken,
+} from '@energinet-datahub/dh/shared/environments';
+
+@Injectable({ providedIn: 'root' })
+export class DhApplicationInsights {
+  private angularPlugin = new AngularPlugin();
+  private appInsights = new ApplicationInsights({
+    config: {
+      instrumentationKey:
+        this.dhAppConfig.applicationInsights.instrumentationKey,
+      extensions: [this.angularPlugin],
+      extensionConfig: {
+        [this.angularPlugin.identifier]: {
+          router: this.router,
+        },
+      },
+    },
+  });
+
+  constructor(
+    private router: Router,
+    @Inject(dhAppEnvironmentToken)
+    private dhAppConfig: DhAppEnvironmentConfig
+  ) {}
+
+  /**
+   * Initialize Application Insights
+   */
+  init(): void {
+    if (this.appInsights.appInsights.isInitialized()) {
+      return;
+    }
+
+    this.appInsights.loadAppInsights();
+  }
+
+  trackEvent(name: string): void {
```

Comment:
Consideration: We should properly consider which methods we want to proxy? You can find the list [here](https://github.com/microsoft/ApplicationInsights-JS#sending-telemetry-to-the-azure-portal). Ex. `trackException`and maybe `trackPageView` could be used for overlays?

Reply:
Nice suggestion. We can also expose more methods if there's a specific need/requirement at a later point. Will add `trackExceptionand` and `trackPageView` for now.